src/server/graphql/models/Team/teamMutation.js

```diff
@@ -127,10 +139,14 @@ export default {
       const userId = leader.userId;
       requireSUOrSelf(authToken, userId);
       // can't trust the client
-      const verifiedLeader = {...leader, isActive: true, isLead: true, isFacilitator: true};
-      await r.table('TeamMember').insert(verifiedLeader);
-      await r.table('Team').insert(team);
-      await r.table('User').get(userId).update({isNew: false});
+      const verifiedLeader = {...leader, isActive: true, isLead: true, isFacilitator: true, checkInOrder: 0};
```

Comment:
the leader should be the first person to check in for the first meeting (show em how it's done!)